packages/common/mui/src/lib/area-chart-header/area-chart-header.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactElement } from 'react';
+import { H4 } from '../typography/typography';
+import { Box, styled, useTheme } from '@mui/material';
+
+export interface AreaHeaderData {
+  leftHeader?: ReactElement<any, any>;
+  legendOne?: ReactElement<any, any>;
+  legendTwo?: ReactElement<any, any>;
+}
+
+export const AreaChartHeader = ({
+  leftHeader,
+  legendOne,
+  legendTwo,
+}: AreaHeaderData) => {
+  const theme = useTheme();
+
+  return (
+    <HeadingContainer>
+      {leftHeader && <H4>{leftHeader}</H4>}
+
+      <Legend>
+        {legendOne && (
+          <LegendItem
+            inline
+            sx={{
+              borderColor: theme.palette.primary.light,
+              borderStyle: 'solid',
+            }}
+          >
+            {legendOne}
+          </LegendItem>
+        )}
+
+        {legendTwo && (
+          <LegendItem
+            inline
+            sx={{
+              borderColor: theme.palette.success.main,
```

Comment:
Do we need to support different colors in the legend here? I think some lines are red

Reply:
I know this is the case but currently we don't have data for a chart that requires this. So I planned that it would be a future add on so I could get this PR out.